runners/flink/src/main/java/org/apache/beam/runners/flink/TestFlinkRunner.java

```diff
@@ -32,8 +32,6 @@
   private FlinkRunner delegate;
 
   private TestFlinkRunner(FlinkPipelineOptions options) {
-    // We use [auto] for testing since this will make it pick up the Testing ExecutionEnvironment
```

Comment:
This works without setting a default because the runner falls back to "auto" internally as well.